Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM python:3.6.4-alpine3.7
```

Comment:
Have test our application with python version 3.7 first?

Reply:
I have run consumer-handler-service and order-email-service with this image some weeks and so far everything looks good